### PR TITLE
add hanasa sub domain

### PR DESCRIPTION
**Pull request by nnivxix**

### CODE REVIEW

**Comment by nnivxix** [comment]
many people still misspelling my name, **"ha na sa"** not ~~"ha sa na"~~. Hahaha

**Comment by fransallen** [comment]
Before I merge this PR. Please:
1. Edit `hanasa.vercel.app/` to `hanasa.vercel.app`.
2. Run `npm run sort` to sort json.

**Comment by fransallen** [comment]
> many people still misspelling my name, "ha na sa" not "ha sa na"

Kalau "hosanna" nanti malah pada nyanyi lagu rohani mas. Sorry for joke :D

**Comment by nnivxix** [comment]
> Before I merge this PR. Please:
> 
> 1. Edit `hanasa.vercel.app/` to `hanasa.vercel.app`.
> 2. Run `npm run sort` to sort json.

done

File: subdomains.json [hunk 1]
```diff
@@ -62,6 +61,7 @@
   "gnataraj": "gnataraj.github.io",
   "goose": "scalgoon.github.io",
   "gski": "matt-gski.github.io",
+  "hanasa": "hanasa.vercel.app/",
```
Comment (fransallen) [inline subdomains.json:64]: Can you remove the trailing slash?

**Review by fransallen — CHANGES_REQUESTED** [review]

File: subdomains.json (hunk 1 above)
Comment (nnivxix) [reply subdomains.json]: yeah, I'am

**Comment by fransallen** [comment]
Merged :confetti_ball:

**Pull request merged**